examples/timeline/other/verticalScroll.html

```diff
@@ -0,0 +1,118 @@
+<html>
+<head>
+  <title>Timeline | Vertical Scroll Option</title>
+
+  <script src="../../../dist/vis.js"></script>
+  <link href="../../../dist/vis.css" rel="stylesheet" type="text/css" />
+
+
+  <style type="text/css">
+    body {
+      color: #4D4D4D;
+      font: 10pt arial;
+    }
+  </style>
+  <script src="../../googleAnalytics.js"></script>
+</head>
+
+<body onresize="/*timeline.checkResize();*/">
+
+<h1>Timeline vertical scroll option</h1>
+
+<div id="mytimeline"></div>
+<br>
+
+<script>
+  function showVisibleItems() {
+    var a = timeline.getVisibleItems();
+    console.log(a);
+    document.getElementById("visibleItemsContainer").innerHTML = ""
+    document.getElementById("visibleItemsContainer").innerHTML += a;
+  };
+
+  // get selected item count from url parameter
+  var count = 1000;
+
+  // create groups
+  var groups = new vis.DataSet([
+    {id: 1, content: 'Truck&nbsp;1'},
+    {id: 2, content: 'Truck&nbsp;2'},
+    {id: 3, content: 'Truck&nbsp;3'},
+    {id: 4, content: 'Truck&nbsp;4'},
+    {id: 5, content: 'Truck&nbsp;5'},
+    {id: 6, content: 'Truck&nbsp;6'},
+    {id: 7, content: 'Truck&nbsp;7'},
+    {id: 8, content: 'Truck&nbsp;8'},
+    {id: 9, content: 'Truck&nbsp;9'},
+    {id: 10, content: 'Truck&nbsp;10'},
+    {id: 11, content: 'Truck&nbsp;11'},
+    {id: 12, content: 'Truck&nbsp;12'},
+    {id: 13, content: 'Truck&nbsp;13'},
+    {id: 14, content: 'Truck&nbsp;14'},
+    {id: 15, content: 'Truck&nbsp;15'},
+    {id: 16, content: 'Truck&nbsp;16'},
+    {id: 17, content: 'Truck&nbsp;17'},
+    {id: 18, content: 'Truck&nbsp;18'},
+    {id: 19, content: 'Truck&nbsp;19'},
+    {id: 20, content: 'Truck&nbsp;20'},
+    {id: 21, content: 'Truck&nbsp;21'},
+    {id: 22, content: 'Truck&nbsp;22'},
+    {id: 23, content: 'Truck&nbsp;23'},
+    {id: 24, content: 'Truck&nbsp;24'},
+    {id: 25, content: 'Truck&nbsp;25'},
+
+  ]);
+
+  // create items
+  var items = new vis.DataSet();
+
+  var order = 1;
```

Comment:
Better use `order` and `truck` as the loop variables, than you don't need `j`and `i`.